benchmark/run_trex.sh

```diff
@@ -24,14 +24,19 @@ args=(
 )
 get_args "$@"
 
+$verbose && echo "Running T-REX .."
+cmd="${trex_install_dir}/hgt3.4 -inputfile=${input_file_nwk}"
+$verbose && echo "Command:"$'\n'"  $cmd"
+
+printf "#TREX\n" >> $output_fp
```

Comment:
@tkosciol I was about to change, but I noticed some issues. See this [post](http://stackoverflow.com/questions/525872/echo-tab-characters-in-bash-script) for people who prefer `printf` over `echo`.
For example, originally it was:
```
printf "$i\t$gene_number\t" >> $output_fp
```
If I change it into echo, it becomes:
```
echo $i$'\t'$gene_number$'\t' >> $output_fp
```
Sounds bit awkward. What do you think?